docs/pages/router/features/head.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Metadata
+---
+
+> **warning**This guide requires `expo@49.0.0-alpha.3` or greater––everything listed here is still experimental. You may need to [use Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing) to use these features.
+
+The Expo `<Head />` component provides a simple and flexible way to manage the meta tags and other important elements of the page head in a universal Expo application. The Expo Head component is designed to be used in both web and mobile applications built with Expo Router. By using this component, developers can easily enhance the search engine optimization (SEO) of their applications.
+
+Native support can be found in the [handoff guide](/router/features/handoff).
+
+## Usage
+
+Be sure to use the `expo-router` Config Plugin if you're planning to use on iOS:
+
+```json title=app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-router",
+        {
+          // Set this to the URL where your iOS universal links can be accessed.
+          // For example, if I build an app "Bacon Blog" and host at `evanbacon.dev`
+          // then `origin` should be `https://evanbacon.dev`.
+          "origin": "<PRODUCTION_URL>"
+        }
+      ]
+    ]
+  }
+}
+```
+
+The `<Head />` component can be imported in any route in the `app` directory, import using `expo-router/head`:
+
+```tsx title=app/index.js
+import { Text } from 'react-native';
+import Head from 'expo-router/head';
+
+export default function App() {
+  return (
+    <>
+      <Head>
+        <title>My App</title>
+        <meta name="description" content="My App description" />
+      </Head>
+      <Text>Hello World</Text>
+    </>
+  );
+}
+```
+
+This renders to the following (simplified):
+
+```html title=dist/index.html
+<html>
+  // highlight-next-line
+  <head>
+    // highlight-next-line
+    <title>My App</title>
+    // highlight-next-line
+    <meta name="description" content="My App description" />
+    // highlight-next-line
+  </head>
```

Comment:
We are using different syntax for code highlight and inline code tips (and it's based on the file typ/extension), in this case this one should work:
* https://github.com/expo/expo/blob/f5d88a9c66baaa565530a409bf72dc9bf9e1ea94/docs/pages/guides/deep-linking.mdx?plain=1#L224-L226